db/deploy/trigger_functions/update_timestamps.sql

```diff
@@ -0,0 +1,57 @@
+-- Deploy connectivity-intake:trigger_functions/update_timestamps to pg
+-- requires: schemas/private
+
+begin;
+
+create or replace function connectivity_intake_public.update_timestamps()
+  returns trigger as $$
+    declare
+      user_sub uuid;
+      owner_id uuid;
+    begin
+      user_sub := (select sub from connectivity_intake_private.session());
+      owner_id := (select owner from connectivity_intake_public.applications where connectivity_intake_public.applications.owner = user_sub);
+      if tg_op = 'INSERT' then
+        if to_jsonb(new) ? 'created_at' then
+          new.created_at = now();
+          new.created_by = owner_id;
```

Comment:
These should probably all record provenance of user making the operation

```suggestion
          new.created_by = user_sub;
```